.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
Should we run `npm run precommit` instead to apply prettier configuration too?
see https://github.com/ecadlabs/taquito/blob/master/packages/taquito-beacon-wallet/package.json#L37

Reply:
sure, that  sounds good. Do you think we should run both lint and precommit? Or will only precommit be suffecient?

Reply:
I think precommit will be sufficient as it will call `eslint --fix`

Reply:
gotcha, I'll remove the lint stage then